web/tests/copy_and_paste.test.js

```diff
@@ -40,6 +40,8 @@ const createPasteEvent = function () {
 
 run_test("paste_handler", () => {
     page_params.development_environment = true;
+
+    // Pasting from another Zulip message
```

Comment:
It's be really helpful to have expanded versions of comments like this before each example; the `input` is basically unreadable. So for example, this one could be "Pasting from another Zulip message containing bold syntax."